pkg/templates/kubeadm/v1beta1/kubeadm.go

```diff
@@ -118,6 +118,8 @@ func NewConfig(ctx *util.Context, host kubeoneapi.HostConfig) ([]runtime.Object,
 		ClusterName: cluster.Name,
 	}
 
+	// clusterConfig.APIServer.ExtraArgs["bind-address"] = hostAdvertiseAddress
```

Comment:
Why is this commented? Should this be removed?

Reply:
it's commented out because it causes incorrect rendered config and apiservers (on nodes 2 and 3) crashloop, but it's worth to return to this back and fix at some point, and be explicit about where to listen.

For example in packet/do(and linode in the future) case machines have both private and public interfaces, provided that there is an LB, there is no single reason to listen on public interfaces.

So, this comment is a leftover after failed attempt to address this, and a reminder to return to this problem someday.

Reply:
It would be nice to add a `TODO` comment and create an issue based on this.

Reply:
done